<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_task_popup</class>
 <widget class="QWidget" name="add_task_popup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>467</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>467</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(16,16,16);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="name_layout">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_edit">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="description_layout">
     <item>
      <widget class="QLabel" name="description_label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Description: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description_edit">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="due_layout">
     <item>
      <widget class="QLabel" name="due_label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Due: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="repeatable_layout">
     <item>
      <widget class="QLabel" name="repeatable_label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Repeatable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="rep_switch" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="repeatable_widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="at_layout">
        <item>
         <widget class="QLabel" name="at_layout_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>At: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="at_timeedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="evety_layout">
        <item>
         <widget class="QLabel" name="every_label">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Every </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="every_box">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Day</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Few Days</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Week</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mounth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Year</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="every_stack">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgba(255, 255, 255, 0);
border: 2px solid white;
border-radius: 25px;
}</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="day">
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </widget>
        <widget class="QWidget" name="few_days">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="few_days_label">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>How many days? </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="few_days_edit">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="week">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="mon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Mon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Wed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="thu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Thu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fri">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Fri</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sun">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 2px solid white;
border-radius: 25px;</string>
            </property>
            <property name="text">
             <string>Sin</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mounth">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="day_edit_2"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="year">
         <widget class="QComboBox" name="mounth_edit">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>10</y>
            <width>136</width>
            <height>51</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QLineEdit" name="day_edit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>200</width>
            <height>51</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="diff_layout">
     <item>
      <widget class="QLabel" name="diff_label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Difficulty: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="diff_box">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Free</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Easy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hard</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="categ_label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Category: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="difficulty_combobox">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Sport</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Education</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Social</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chores</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Meal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Higyene</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Job</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mental</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="submit_button">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(40,40,40);</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
